lib/semantics/check-declarations.cpp

```diff
@@ -293,11 +294,56 @@ void CheckHelper::CheckValue(
   }
 }
 
+void CheckHelper::CheckAssumedTypeEntity(  // C709
+    const Symbol &symbol, const ObjectEntityDetails &details) {
+  if (const DeclTypeSpec * type{symbol.GetType()};
+      type && type->category() == DeclTypeSpec::TypeStar) {
+    if (!symbol.IsDummy()) {
+      messages_.Say("Assumed-type entity '%s' must be a dummy "
+                    "argument"_err_en_US,
```

Comment:
This message fits on one line without splitting it.

Reply:
It looks to me like the line is 82 characters long if I merge the strings.  I'll leave it as is.

Reply:
If you merge the strings and reformat it fits.

Reply:
Ahh...  Thanks!

Reply:
There are some other similar strings.  I'll fix them up, too.